src/middle-pgsql.cpp

```diff
@@ -28,7 +28,10 @@
 #include <osmium/memory/buffer.hpp>
 #include <osmium/osm/types_from_string.hpp>
 
+#include <nlohmann/json.hpp>
```

Comment:
This library is not documented in the readme as being required, resulting in compilation errors.

Reply:
I have added it to the README and also to the CMake config.